src/sql/workbench/services/notebook/node/localContentManager.ts

```diff
@@ -22,6 +22,29 @@ import { nbformat } from 'sql/parts/notebook/models/nbformat';
 type MimeBundle = { [key: string]: string | string[] | undefined };
 
 export class LocalContentManager implements nb.ContentManager {
+
+	public async loadFromContentString(contentString: string): Promise<nb.INotebookContents> {
```

Comment:
TODO refactor name and this overall API to have a cleaner design / naming convention.

Reply:
Tests are failing in notebookModel.test since we changed the entire pattern of loading content. Will fix them in next commit.